CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 #    - Python_INCLUDE_DIR "Path to the python include files (e.g., /path/to/include/python3.7)"
 #    - SPHINX_ROOT "Root directory for Sphinx: 'bin/sphinx-build' (or similar) should be in this dir."
 #
-#    For any ${AddOn} of: ambit, CheMPS2, dkh, libefp, erd, gau2grid, gdma, Libint2, PCMSolver, pybind11, pylibefp,
+#    For any ${AddOn} of: ambit, CheMPS2, dkh, ecpint, libefp, erd, gau2grid, gdma, Libint2, PCMSolver, pybind11, pylibefp,
```

Comment:
Confused about the nomenclature here. `ecpint` as opposed to `libecpint`, but then there's `libefp`, `libint2`...

Reply:
yeah, it's largely determined by what ppl define for `project(my_Name)` in cmake. after that, cmake is highly patterned on that word (including the capitalization). The libecpint project isn't entirely consistent on this, but if I can get away with "ecpint" (and I seem to be able to so far), I think that's preferable. other opinions?

Reply:
I prefer `libecpint` purely because that's the name of the repo. I'm not sure @robashaw has an opinion?

Reply:
he's using `ecpint` for the `find_package` name and the namespace
https://github.com/robashaw/libecpint/blob/master/src/CMakeLists.txt#L81-L92 so sticking with that for now. can still be rejiggered after merge if Shaw wants to make a ruling.

Reply:
This is my fault for making a dumb decision early on (back when I didn't understand CMake at all), but yes I think it's safest keeping everything as "ecpint" to be consistent. Also I've seen the PR on libecpint and I'll hopefully get a chance to check and merge it soon.